lib/helpers/project-init.js

```diff
@@ -33,7 +33,7 @@ module.exports = function(options) {
   var presets = initPresets[target]();
   var projectName = (target === 'addon') ? 'my-addon' : 'my-app';
   var cliOptions = {
-    package: options.package || path.resolve(process.cwd(), '..', '..', 'tests', 'fixtures', type, 'package'),
+    package: options.package || path.resolve(process.cwd(), '..', '..', 'node-tests', 'fixtures', type, 'package'),
```

Comment:
Okay, so I completely forgot that I'm already using findup here: https://github.com/ember-cli/ember-cli-blueprint-test-helpers/blob/master/lib/helpers/blueprint-helper.js#L204

The line we've been discussing is simply a backup, and is only used if someone were to use the project init helper on it's own. The project root is passed in from the blueprint helper, so we don't really need to worry about the backup in our use case.

Sorry I didn't catch this earlier.